src/WebformCivicrmPostProcess.php

```diff
@@ -2511,7 +2511,7 @@ private function fillDataFromSubmission() {
                 $n = $c;
               }
             }
-            $this->data[$ent][$c][$ent][$n][$name] = $customValue ?? $val;
+            $this->data[$ent][$c][$ent][1][$name] = $customValue ?? $val;
```

Comment:
ah no - i meant to change the above assignment of 

    $n = $c;

 to 

    $n = 1;

instead. The current change will affect all other entities as well. Sorry if that wasn't clear @MegaphoneJon.